addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -687,6 +677,28 @@ async function toInline($editable, cssRules, $iframe) {
     flattenBackgroundImages(editable);
     formatTables($editable);
     responsiveToStaticForOutlook(editable);
+    // Fix Outlook image rendering bug.
```

Comment:
This comment screams for more information. Which bug ? A future developer looking to revert this commit for some reason has no idea what it should look for to test against the original bug. The commit message doesn't help either.

Reply:
My bad, I thought it was a new comment.